nips/06.md

```diff
@@ -0,0 +1,15 @@
+NIP-06
+======
+
+Basic key derivation from mnemonic seed phrase
+----------------------------------------------
+
+`draft` `optional` `author:fiatjaf`
+
+[BIP39](https://bips.xyz/39) is used to generate mnemonic seed words and derive a binary seed from them.
+
+[BIP32](https://bips.xyz/32) is used to derive the path `m/44'/1237'/ 0'/ 0'` (according to the Nostr entry on [SLIP44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)).
```

Comment:
I think this should be m/44'/1237'/0'/0/0

Reply:
Suggestion denied.

Reply:
Ok, I screwed up. My implementation was broken, therefore everybody will lose their mnemonics anyway. So I'll change the spec to this.